src/tracer.js

```diff
@@ -188,6 +188,8 @@ export default class Tracer {
    *        the created Span object. The time should be specified in
    *        milliseconds as Unix timestamp. Decimal value are supported
    *        to represent time values with sub-millisecond accuracy.
+   * @param {number} [options.traceId128bit] - generate root span with a
```

Comment:
I believe this is a boolean based on the config and a test, so it should be `@param {boolean} ...` instead